synapse/cortex.py

```diff
@@ -578,12 +578,12 @@ async def delStormDmon(self, iden):
         return await self.cell.delStormDmon(iden)
 
     @s_cell.adminapi(log=True)
-    async def enableMigrationMode(self):
-        await self.cell._enableMigrationMode()
+    async def enableMigrationMode(self): # pragma: no cover
+        s_common.deprdate('CoreApi.enableMigrationMode', '2024-05-05')
```

Comment:
Should we allow these to keep working until the deprecation date?

Reply:
I guess we can't really if we're ripping out the stuff that makes it work.

Reply:
I can't imagine anybody is using them, but we could if you feel like we should :)